yeat/workflows/snakefiles/Pacbio

```diff
@@ -124,3 +124,22 @@ rule unicycler:
         unicycler -l {input.reads} -t {threads} -o {params.outdir} {params.extra_args}
         ln -s assembly.fasta {output.contigs}
         """
+
+
+rule metamdbg:
+    output:
+        contigs="analysis/{sample}/{label}/metamdbg/{sample}_contigs.fasta"
+    input:
+        reads="seq/input/{sample}/combined-reads.fq.gz"
+    conda:
+        "metaMDBG"
```

Comment:
See #57 . 

TLDR: `metaMDBG` packages are not compatible with YEAT's environmental packages. Activating `metaMDBG` environment whenever the rule `metamdbg` is called.